pliers/extractors/text.py

```diff
@@ -402,3 +402,33 @@ def _extract(self, stim):
 
         return ExtractorResult(features_list, stim, self,
                                features=self.features, orders=order_list)
+
+class WordCounterExtractor(TextExtractor):
+
```

Comment:
We have a convention of tracking any initialization parameters we want to log (basically, anything that helps the user understand the instance's expected behavior) using a `_log_attributes` class variable. So you could add the line `_log_attributes = ('lemmatize',)` here. See other classes in the module for examples.